packages/manager/src/managers/prismicRepository/types.ts

```diff
@@ -167,3 +167,32 @@ export const Environment = t.type({
 	),
 });
 export type Environment = t.TypeOf<typeof Environment>;
+
+export const OnboardingStepId = t.union([
+	t.literal("createPrismicProject"),
+	t.literal("chooseLocale"),
+	t.literal("createProject"),
+	t.literal("setupSliceMachine"),
+	t.literal("createPageType"),
+	t.literal("codePage"),
+	t.literal("createSlice"),
+	t.literal("reviewAndPush"),
+	t.literal("createContent"),
+	t.literal("renderPage"),
+]);
```

Comment:
I'm not sure what is the best option:
- on one hand, if the content is updated we have to update the types (I could add a comment about that next to the content to remind of that)
- on the other, the steps will be toggled in other parts of the application using step id, having those types will help prevent any typos etc.

I'm more in favour of the first version since updating content would require changes in code anyway. And hopefully the content wouldn't be updated too often (if at all).

Reply:
⚠️ Here for me, it's also a product question. In the case we are receiving something unknown, how do we want to react? Hiding the onboarding card? Only displaying the known steps? etc

Reply:
So it should just not fail if a step is not known, just not displaying it, right?

Reply:
Correct

Reply:
Removed this